ansible/tasks/postgres-extensions/21-pg_jsonschema.yml

```diff
@@ -0,0 +1,51 @@
+# pg_jsonschema
+- name: pg_jsonschema - download & install dependencies
+  apt:
+    pkg:
+      - make
+      - gcc
+      - pkg-config
+      - clang
+      - postgresql-server-dev-{{ postgresql_major }}
+      - libssl-dev
+    update_cache: yes
+    install_recommends: no
+
+- name: check if cargo is installed
+  shell: command -v cargo
+  register: cargo_exists
+  ignore_errors: yes
+
+- name: download rust/cargo installer
+  when: cargo_exists is failed
+  get_url:
+    url: https://sh.rustup.rs
+    dest: /tmp/sh.rustup.rs
+    mode: '0755'
+    force: 'yes'
+  tags:
+    - rust
+
+- name: install rust/cargo
+  when: cargo_exists is failed
+  shell: /tmp/sh.rustup.rs -y
+  tags:
+    - rust
+
+- name: install cargo-pgx
```

Comment:
Have tested and it looks to be working. Only hitch is that we would have to temporarily grant the `postgres` Linux user ownership of `/usr/lib/postgresql/lib` to ensure `cargo pgx install --release` is fully installed. Else, the following error would be returned:
```
Error: 
   0: failed copying `/tmp/pg_jsonschema/target/release/libpg_jsonschema.so` to `/usr/lib/postgresql/lib/pg_jsonschema.so`
   1: Permission denied (os error 13)

Location:
   /home/postgres/.cargo/registry/src/github.com-1ecc6299db9ec823/cargo-pgx-0.4.5/src/command/install.rs:234

  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ SPANTRACE ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

   0: cargo_pgx::command::install::install_extension with pg_version=14.14 release=true test=false features=["pg14"]
      at /home/postgres/.cargo/registry/src/github.com-1ecc6299db9ec823/cargo-pgx-0.4.5/src/command/install.rs:84
   1: cargo_pgx::command::install::execute
      at /home/postgres/.cargo/registry/src/github.com-1ecc6299db9ec823/cargo-pgx-0.4.5/src/command/install.rs:51
```

Should be okay since we're returning ownership to the `root` Linux user afterward. @olirice which OS user do you use for these steps when following the installation in the [pg_jsonschema readme](https://github.com/supabase/pg_jsonschema#installation)?

Reply:
The installation instructions in the readme are more for trying it vs a release so the OS user can be the user who ran `cargo pgx init` when setting up the `pgx` framework

> tested running everything as the postgres Linux user and it looks to be working. Only hitch is that we would have to temporarily grant the postgres Linux user ownership of /usr/lib/postgresql/lib to ensure cargo pgx install --release is fully installed

Could you please please push that change to this branch? I'm not clear how to do it